src/Facebook/InstantArticles/Client/Client.php

```diff
@@ -103,10 +103,26 @@ public function importArticle($article, $published = false)
           'development_mode' => $this->developmentMode,
         ]);
 
+        // Re-scrape Graph object for article URL
```

Comment:
Consider checking a parameter (add a new one on the importArticle method) that can be default to false.
This parameter could be: $forceRescrape defaults to false.
it would be:
```
   if ($forceRescrape) {
      // do the rescrape
   } 
```